Cdnfsd_TransparentCDN_Page.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace W3TC;
+
+
+class Cdnfsd_TransparentCDN_Page {
+	// called from plugin-admin
+	static public function admin_print_scripts_performance_page_w3tc_cdn() {
+		wp_enqueue_script( 'w3tc_cdn_transparentcdn_fsd',
+			plugins_url( 'Cdnfsd_TransparentCDN_Page_View.js', W3TC_FILE ),
+			array( 'jquery' ), '1.0' , true);
+	}
+
+	static public function print_dummy_text() {
+		wp_enqueue_script ( 'w3tc_cdn_transparent_dummytext', 
```

Comment:
In order to inject localized data into a script, you must first register the script, localize to add the data, and then enqueue the script.  Please see https://developer.wordpress.org/reference/functions/wp_localize_script/#user-contributed-notes